loadtest/store/memstore/store.go

```diff
@@ -30,16 +30,20 @@ type MemStore struct {
 
 // New returns a new instance of MemStore.
 func New() *MemStore {
-	return &MemStore{
-		posts:          map[string]*model.Post{},
-		teams:          map[string]*model.Team{},
-		channels:       map[string]*model.Channel{},
-		channelMembers: map[string]map[string]*model.ChannelMember{},
-		teamMembers:    map[string]map[string]*model.TeamMember{},
-		users:          map[string]*model.User{},
-		reactions:      map[string][]*model.Reaction{},
-		roles:          map[string]*model.Role{},
-	}
+	newStore := MemStore{}
```

Comment:
Nit: Let's do `newStore := &MemStore{}` and remove the & from the return statement. Also rename newStore to s, because it's just 3 lines.